extensions/geocoder/deployment/src/main/java/org/apache/camel/quarkus/component/geocoder/deployment/GeocoderProcessor.java

```diff
@@ -50,6 +50,7 @@ ExtensionSslNativeSupportBuildItem activateSslNativeSupport() {
         items.add(new ReflectiveClassBuildItem(false, true, "com.google.maps.model.Bounds"));
         items.add(new ReflectiveClassBuildItem(false, true, "com.google.maps.model.LatLng"));
         items.add(new ReflectiveClassBuildItem(false, true, "com.google.maps.model.LocationType"));
+        items.add(new ReflectiveClassBuildItem(false, true, "com.google.maps.GeoApiContext$Builder"));
```

Comment:
This is only needed for for the reflection hack in the test, right? I guess it is not necessary for users normal runtime operation?

If so, we can use a configuration property in the test:

```
quarkus.camel.native.reflection.include-patterns=com.google.maps.GeoApiContext$Builder
```

Reply:
yes!! I'll do that !!

Reply:
done !!